docs/ci-cd-environment/setup-caching-on-aws-s3.md

```diff
@@ -0,0 +1,164 @@
+---
+description: This guide describes how to setup the cache CLI to work with an AWS S3 bucket
+---
+
+# Setting up caching with AWS S3
+
+In order to use S3 as a storage backend, the cache CLI needs access only to perform a few actions on an S3 bucket in your AWS account.
```

Comment:
As a precaution I'd add a mention that this is in the context of s/h. Just in case someone opens this page looking for default cloud caching.